CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
```

Comment:
Should we convert layout-large & friends to use `h*dp` instead? I'm concerned with the inconsistency. Since we're not changing FFES too much in the short term, maybe this is best left for a follow-up? I'm not sure.

Reply:
I think it's reasonable to change them now.  I can take care of that in this PR.